apps/wallet/tests/balanceChanges.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+import { generateAddress, generateKeypairFromMnemonic } from './utils/localnet';
+
+const mnemonic = [
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+    'beef',
+];
+
+test('send 300 SUI and view transaction activity', async ({
+    page,
+    extensionUrl,
+}) => {
+    const keypair = await generateKeypairFromMnemonic(mnemonic.join(' '));
+    const address = generateAddress(keypair);
+
+    await createWallet(page, extensionUrl);
+
+    await page.getByTestId('faucet-request-button').click();
```

Comment:
You probably want two tests here: one to programmatically invoke (to test external transactions), one that uses the button

Reply:
done